lib/sandbox/legacy-execute.js

```diff
@@ -48,6 +48,18 @@ module.exports = {
                         sealed = true;
 
                         err && bridge.dispatch('error.execution', cursor, err, id);
+
+                        // note - this is a hack to work around a limitation of CircularJSON, which strips all
+                        // properties that have the value "undefined". This interferes with the tests object, which
+                        // relies on the reference being there to mark failed tests.
+                        if (options && options.globals && options.globals.tests) {
```

Comment:
rename `t` to `prop` and move the variable instantiation to the beginning of this function please.